src/commands/logging/cmd.js

```diff
@@ -24,7 +24,7 @@ export function setup() {
             storage.session.setLogApiKey(key);
             storage.session.setLogApiSecret(secret);
             console.log("Listing available ID Cloud log sources...");
-            const conn = getConnection(storage.session.getTenant());
+            const conn = await getConnection(storage.session.getTenant());
```

Comment:
there is another call to getConnection() on line 65 and the getConnection() method no longer takes params.